webui-vue/src/css/quasar.variables.scss

```
@@ -0,0 +1,30 @@
+// BAREOS® - Backup Archiving REcovery Open Sourced
+//
```

Stylelint (17.7.0)

[error] Unexpected empty comment (scss/comment-no-empty)

(scss/comment-no-empty)